### PR TITLE
formatting with prettier

### PR DESCRIPTION
**Pull request by avi312NHS**
## Type of changes

- [ ] Refactoring (non-breaking change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)

## Checklist

- [ ] I am familiar with the [contributing guidelines](../docs/CONTRIBUTING.md)
- [ ] I have followed the code style of the project
- [ ] I have added tests to cover my changes
- [ ] I have updated the documentation accordingly
- [ ] This PR is a result of pair or mob programming

---

## Sensitive Information Declaration

To ensure the utmost confidentiality and protect your and others privacy, we kindly ask you to NOT including [PII (Personal Identifiable Information) / PID (Personal Identifiable Data)](https://digital.nhs.uk/data-and-information/keeping-data-safe-and-benefitting-the-public) or any other sensitive data in this PR (Pull Request) and the codebase changes. We will remove any PR that do contain any sensitive information. We really appreciate your cooperation in this matter.

- [ ] I confirm that neither PII/PID nor sensitive data are included in this PR and the codebase changes.

### CODE REVIEW

File: scripts/config/.pre-commit.yaml [hunk 1]
```diff
@@ -1,36 +1,36 @@
 repos:
```
Comment (andrew-cleveland) [inline scripts/config/.pre-commit.yaml:1]: Has this been tested? adding indents in a yaml file can break it so we need to be sure its not causing problems

Comment (avi312NHS) [reply scripts/config/.pre-commit.yaml:1]: ok ill just do the JS files for now then, yes good point about .yaml files

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,4 +1,5 @@
 <!-- markdownlint-disable-next-line first-line-heading -->
+
```
Comment (andrew-cleveland) [inline .github/PULL_REQUEST_TEMPLATE.md:2]: This is failing the markdown check

**Comment by avi312NHS** [comment]
closing with new PR for just JS and TS files

**Pull request closed without merging**